### PR TITLE
UX: use transparent giphy logo for light themes

### PR DESCRIPTION
**Pull request by gabenp**
This logo looked really ugly on light themes:

image

With this change it could look like:

image

### CODE REVIEW

**Comment by CvX** [comment]
Can you use this version instead? I resized it (50%) and ran through ImageOptim. So instead of a ~1.1MB it's ~0.28MB:

PoweredBy_200_Horizontal_Light-Backgrounds_With_Logo

File: common/common.scss [hunk 1]
```diff
@@ -94,7 +94,6 @@ html:not(.discourse-gifs-with-img)
     flex-flow: row-reverse;
     img {
       height: 42px;
```
Comment (CvX) [inline common/common.scss:96]:
```suggestion
      height: 30px;
```

File: common/common.scss [hunk 2]
```diff
@@ -94,7 +94,6 @@ html:not(.discourse-gifs-with-img)
...
-      background-color: black;
       border-radius: 3px;
       padding: 4px;
```
Comment (CvX) [inline common/common.scss:98]:
```suggestion
```

Comment (CvX) [inline common/common.scss:98]:
```
height: 30px
```

we don't need padding and rounded corners here

File: common/common.scss [hunk 3]
```diff
@@ -94,9 +94,6 @@ html:not(.discourse-gifs-with-img)
     flex-flow: row-reverse;
     img {
       height: 42px;
```
Comment (CvX) [inline common/common.scss:96]:
```suggestion
      height: 30px;
```

To make it exactly half the hight of the file.

**Review by CvX — APPROVED** [review]

File: common/common.scss (hunk 3 above)
Comment (gabenp) [reply common/common.scss:96]: Oops I misunderstood your previous comment, gotcha :+1:

**Pull request merged**